exosphere-shared/src/docker-helper.ls

```diff
@@ -43,14 +44,19 @@ class DockerHelper
     child_process.exec-sync "docker rm -f #{container}" if @container-exists container
 
 
-  @run-image = (container, image) ->
-    if container is \test-mongo
-      child_process.exec-sync "docker run -d --name=#{container} -p 27017:27017 #{image}"
+  @run-image = (container, done) ~>
+    if container.container-name is \test-mongo
```

Comment:
Why do we have special handling for `test-mongo` here? Can't we start it using `ObservableProcess` below?